pkg/crd/crd.go

```diff
@@ -148,6 +156,43 @@ func GenerateCRDs(caBundle string, reference apiextv1.ServiceReference) []crder.
 				IsNamespaced(true).
 				AddVersion("v1", &v1.DynamicBindConfiguration{}, nil)
 		}),
+		hobbyfarmCRD(&v1.Scope{}, func(c *crder.CRD) {
+			c.IsNamespaced(true).AddVersion("v1", &v1.Scope{}, func(cv *crder.Version) {
+				cv.WithColumn("Name", ".displayName").IsServed(true).IsStored(true)
```

Comment:
i would rename this column to "Display Name", otherwise it can be confused with the default column "Name" that prints the metadata.name

Same for the Setting CRD